packages/ethereum-erc20-swap-provider/contracts/HTLC.sol

```diff
@@ -65,4 +71,4 @@ contract HTLC {
       }
     }
   }
-}
+}
```

Comment:
Can you add an empty line to the end of this file?

Reply:
Yes